tb_plugin/torch_tb_profiler/profiler/data.py

```diff
@@ -90,7 +84,9 @@ def parse(run_dir, worker, span, path, caches):
         for data in trace_json:
             event = trace.create_event(data)
             if event is not None:
+                profile.profiler_start_ts = min(profile.profiler_start_ts, event.ts)
```

Comment:
we can get the start time by find the root node in tid2tree, which should be much more faster

Reply:
It does not necessary be faster. Since this is scanned only once in subprocess. You inevitablely scan through all events, Record it there will be much resonable.